MediaBrowser.Api/Library/LibraryService.cs

```diff
@@ -759,13 +759,12 @@ private void LogDownload(BaseItem item, User user, AuthorizationInfo auth)
         {
             try
             {
-                _activityManager.Create(new ActivityLogEntry
+                _activityManager.Create(new Jellyfin.Data.Entities.ActivityLog(
```

Comment:
I'm not particularly thrilled about using DB entities directly in the API layer. Imo the DB should be in a separate project that exposes service classes and DTO models.

@Bond-009 @mark-monteiro @EraYaN thoughts?

Reply:
@cvium Using DTO models would be my preference too, afaik that pattern is still widely used. That said, it might be better to handle in a follow-up PR, since the original code was also not using DTOs and the scope of this PR is already pretty large IMO

Reply:
I agree with Mark here, let's introduce middle-objects in a later PR. I also agree that we need a layer so we can decouple parts more, but this PR is already huge.

Reply:
🙏

Reply:
As long as it is being handled. Preferably sooner rather than later.